File: tests/conformance/fixtures/python/20-edge-duplicate-after.py
from sykli import Pipeline

p = Pipeline()

p.task("build").run("make build")

# Duplicate after calls should be deduplicated
p.task("test").run("go test").after("build").after("build")

p.task("deploy").run("./deploy.sh").after("build", "test", "build", "test")

p.emit()

File: tests/conformance/fixtures/rust/20-edge-duplicate-after.rs
use sykli::Pipeline;

fn main() {
    let mut p = Pipeline::new();
    p.task("build").run("make build");
    p.task("test").run("go test").after(&["build"]);
    p.task("deploy").run("./deploy.sh").after(&["build", "test"]);
    p.emit();
}
